### ISSUE TITLE
issues with fresh conda installation

### ISSUE THREAD

**Issue by ahriley** [adrn/gala#168]
it looks like running a clean installation of `gala` via `conda` fails b/c it's expecting `pytest` to be installed along with it. basic steps go:
```
conda create -n test -c conda-forge astro-gala
...
python
>>> import gala
```
which fails with the following error message
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/__init__.py", line 26, in <module>
    from . import dynamics
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/dynamics/__init__.py", line 2, in <module>
    from .orbit import Orbit, CartesianOrbit
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/dynamics/orbit.py", line 14, in <module>
    from .util import peak_to_peak_period
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/dynamics/util.py", line 6, in <module>
    from astropy.tests.helper import quantity_allclose
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/astropy/tests/helper.py", line 14, in <module>
    import pytest
ModuleNotFoundError: No module named 'pytest'
```
after installing `pytest`, then it breaks trying to find `six` (I think specifically `astropy.extern.six`)
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/__init__.py", line 26, in <module>
    from . import dynamics
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/dynamics/__init__.py", line 3, in <module>
    from .analyticactionangle import *
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/dynamics/analyticactionangle.py", line 13, in <module>
    from ..potential import (Hamiltonian, PotentialBase,
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/potential/__init__.py", line 1, in <module>
    from .potential import *
  File "/Users/alexanderriley/miniconda3/envs/test/lib/python3.8/site-packages/gala/potential/potential/__init__.py", line 2, in <module>
    from .cpotential import *
  File "gala/potential/potential/cpotential.pyx", line 15, in init gala.potential.potential.cpotential
ImportError: cannot import name six
```
which might be leftover from #135

**Comment by adrn** [comment]
Hey @ahriley - This should be fixed on master. Could you try updating to the latest dev version? (there are some API changes with the mock stream functionality, so I would understand if that's not possible for you)
```
pip install git+https://github.com/adrn/gala
```

I'm going to fix this and release a v1.0.1 just in case - will hopefully get to that this afternoon.

**Comment by ahriley** [comment]
thanks @adrn! the `pytest` part is still a problem...not a huge one since installing it explicitly makes everything work fine, but if it's a dependency can it get added as such so that conda/pip will install it automatically?

**Comment by adrn** [comment]
I believe this should be fixed now with v1.1. I just tried both with `pip` and `conda-forge`:

**pip**:
```bash
conda create -n test -y pip
conda activate test
pip install astro-gala
python -c "import gala"
```

**conda**:
```bash
conda create -n test -y pip
conda activate test
conda install -y -c conda-forge astro-gala
conda install -c conda-forge openblas==0.3.3
python -c "import gala"
```

Note: the `openblas` downgrade appears to fix an issue related to [this issue](https://github.com/numpy/numpy/issues/14165) - haven't tracked down where the problem is though

**Comment by adrn** [comment]
Let me know if you still have problems! BTW: for the next release, I'll provide pip wheels as well.
